### PR TITLE
Add tab completion for in program commands

### LINKED ISSUE #783: Tab completion for commands
It would be nice if mps-youtube could allow to expand the commands when TAB key is pressed. 

## Suggestion
As Linux user I am famillar with the tab expansion, which is a nice feature. This feature is somehow missing in mps-youtube and it would be usefull. 

For instance 
`set pl<TAB>`
should display
`set player`
pressing <TAB> second time should display
`set playerargs`

## Your Environment
mpsyt 0.2.8
Android 6.0.1 (CM13)
mpv

### PR DESCRIPTION
**Pull request by vn-ki**
Closes #783.

### CODE REVIEW

File: mps_youtube/util.py [hunk 1]
```diff
@@ -24,6 +24,26 @@
...
+COMMANDS = ['play', 'set', 'album', 'all', 'playurl', 'browserplay',
+            'showconfig', 'encoders', 'dv', 'da', 'dl', 'd', 'download',
+            'dapl', 'dvpl', 'daupl', 'dvupl', 'daurl', 'dvurl', 'mkp',
+            'lastfm_connect', 'rmp', 'add', 'mv', 'save', 'open', 'view',
+            'ls', 'vp', 'clearcache', 'help', 'exit', 'history',
+            'history recent', 'history clear', 'channels', 'user', 'live',
+            'pls', 'mix', 'url', 'url_file', 'pl', 'rm', 'undump', 'dump',
+            'sw', 'shuffle', 'reverse', 'repeat', 'suser', 'splaylist']
+
+SET_COMMANDS = ['set order', 'set user_order', 'set max_results',
```
Comment (ids1024) [inline mps_youtube/util.py]: You should be able to iterate over `config` to get this list.

File: mps_youtube/util.py [hunk 2]
```diff
@@ -24,6 +24,26 @@
 
 XYTuple = collections.namedtuple('XYTuple', 'width height max_results')
 
+COMMANDS = ['play', 'set', 'album', 'all', 'playurl', 'browserplay',
```
Comment (ids1024) [inline mps_youtube/util.py]: This is awkward... I don't like the redundancy, but the current regex method makes it difficult to do anything else...

Comment (vn-ki) [reply mps_youtube/util.py]: I don't think there's an other way to do it.

**Comment by vn-ki** [comment]
@ids1024 Any other changes required?

**Comment by doriancuentas** [comment]
how can I find out why tab completion not working on my configuration?
`
$ mpsyt --version
mpsyt version      : 0.2.8
   notes           : released 17 February 2018
pafy version       : 0.5.4 (youtube-dl backend)
youtube-dl version : 2019.05.20
Python version     : 3.7.3 (default, May 13 2019, 18:48:30) 
[GCC 7.4.0]
Processor          : x86_64
Machine type       : x86_64
Architecture       : 64bit, 
Platform           : Linux-4.18.0-20-generic-x86_64-with-debian-buster-sid
sys.stdout.enc     : UTF-8
default enc        : utf-8
Config dir         : /home/user/.config/mps-youtube
env:TERM           : xterm-256color
env:SHELL          : /bin/bash
env:LANG           : en_GB.UTF-8
env:LANGUAGE       : en_GB:en_US:es:zh_CN
`

**Pull request merged**